test/WebSites/ApiExplorerWebSite/ApiExplorerWebSite.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <Import Project="..\..\..\build\common.props" />
+
+  <PropertyGroup>
+    <TargetFrameworks>net452;netcoreapp1.1</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netcoreapp1.1</TargetFrameworks>
+    <RuntimeIdentifier Condition=" '$(TargetFramework)' != 'netcoreapp1.1' ">win7-x64</RuntimeIdentifier>
```

Comment:
Supposedly, we don't have to do this anymore. Last week the SDK remove the requirement to specify RuntimeIdentifier for .NET Framework projects. Can you try removing this and see if it still works? If not, it may be a bug in the SDK.

Repeat for all test websites below...